### DISCUSSION TITLE
for everyone involved, with love.

### DISCUSSION THREAD

**Discussion by vicksbr** [mantinedev/mantine#1641] [Kind words]
### **Hey you awesome people..**

mantine is my all time favorite ui library and I get so much inspiration from yours implementations and apis!  

thank you so much.

**Comment by rtivital** [comment]
Thanks for kind words, Pedro!

**Reply by Emiltayeb** [reply]
After reading the docs, I felt like I was reading a beautiful book.
Beautiful work, I really like it
